app/initializers/google-drive.js

```diff
@@ -14,5 +14,7 @@ export default {
     application.inject('route', 'documentSource', 'document-source:main');
     application.inject('controller', 'documentSource', 'document-source:main');
     application.inject('adapter:application', 'documentSource', 'document-source:main');
+    application.inject('adapter:google-drive-permission', 'documentSource', 'document-source:main');
+    application.inject('component:document-creator', 'documentSource', 'document-source:main');
   }
-};
\ No newline at end of file
+};
```

Comment:
There were two initializers that were in charge of injecting `documentSource` into parts of the application
* google-drive-permission
* google-drive (this one).

Since the new component also needs access to `documentSource`, that would add a third initializer. Instead, I've moved them all to this one, since it really sounds like something a single entity should be in charge of.